jenkins-design-language/src/js/components/status/StatusIndicator.jsx

```diff
@@ -6,15 +6,17 @@ import SvgStatus from './SvgStatus';
 
 const validResultValues = {
     // Mostly from io/jenkins/blueocean/rest/model/BlueRun.java
+    // TODO: Do we want to move this? I have NFI
```

Comment:
Yeah I put that comment in when I was planning to move the component, but I decided against it because I didn't want to give up Flow. I'll remove it.